project.clj

```diff
@@ -8,5 +8,6 @@
                  [metosin/ring-http-response "0.9.0"]
                  [ring/ring-core "1.6.0"]
                  [threatgrid/ring-graphiql "0.1.1"]
-                 [threatgrid/ring-graphql-voyager "0.1.2"]]
-  :profiles {:test {:dependencies [[ring/ring-mock "0.3.0"]]}})
+                 [viebel/ring-graphql-playground "0.1.1"]]
+  :profiles {:test {:dependencies [[ring/ring-mock "0.3.0"]]}
+             :dev {:dependencies [[ring/ring-mock "0.3.0"]]}})
```

Comment:
Better to leave this untouched and run the repl w/ `lein with-profile +test repl`